apps/nextjs-website/src/components/Editorial.tsx

```diff
@@ -1,9 +1,8 @@
 'use client';
-import { Editorial as EditorialEC } from '@pagopa/pagopa-editorial-components';
-import { EditorialProps } from '@pagopa/pagopa-editorial-components/dist/components/Editorial';
-import { Stack } from '@mui/material';
 import Image from 'next/image';
 import MarkdownRenderer from './MarkdownRenderer';
+import { Editorial as EditorialEC } from '@react-components/components';
```

Comment:
```suggestion
import { Editorial as EditorialRC } from '@react-components/components';
```
Let's keep it consistent.